### ISSUE TITLE
Option to switch UI languages - English/Kannada . And to switch character display captions from Kannada to IAST or any other Indic script

### ISSUE THREAD

**Issue by pluday** [mythicsociety/AksharaBhandara#7]
Option to switch UI languages - English/Kannada . And to switch character display captions from Kannada to IAST or any other Indic script

**Comment by mythicsociety** [comment]
Will this be a site-wide feature? Where a user can switch display language using a dropdown?
This is called I-18 internationalization and we need to add translations for every language that we plan to support. I will prepare a list of text used throughout the site along with their translations and ask you for a review.

**Comment by pluday** [comment]
yes it will be a site wide feature. The idea is we use this same tool for other scripts too in future.
